cupboard_app/test_queries.py

```diff
@@ -88,6 +92,72 @@ def test_get_ingredient(self):
             get_ingredient(name='doesnt_exist')
 
 
+class CustomIngredientQueries(TestCase):
+    ing1 = None
+    ing2 = None
+    user = None
+
+    def setUp(self):
+        self.user = User.objects.create(username='test_user', email='test_user@cupboard.app')
+        self.ing1 = CustomIngredient.objects.create(
+            user=self.user, name='test_ingredient1', type='test_type1'
+        )
+        self.ing2 = CustomIngredient.objects.create(
+            user=self.user, name='test_ingredient2', type='test_type1'
+        )
+
+    def test_create_custom_ingredient(self):
+        """
+        Testing create_custom_ingredient creates a custom ingredient for the user
+        """
+        create_custom_ingredient(username='test_user', name='test_ingredient3', type='test_type2')
+        self.assertEqual(
+            CustomIngredient.objects.filter(
+                user=self.user, name='test_ingredient3', type='test_type2'
+            ).exists(),
+            True
+        )
+        self.assertEqual(len(CustomIngredient.objects.all()), 3)
+
+        create_custom_ingredient(username='test_user', name='test_ingredient3', type='test_type2')
+        self.assertEqual(len(CustomIngredient.objects.all()), 3)
+
+    def test_get_all_custom_ingredients(self):
+        """
+        Testing get_all_custom_ingredients retrieves all the custom ingredients for the user
+        """
+        user2 = User.objects.create(username='test_user2', email='test_user2@cupboard.app')
+        CustomIngredient.objects.create(user=user2, name='test_ingredient3', type='test_type1')
```

Comment:
Is there a reason these two objects were created? Because we are not doing asserts for these two.

If you want to check that even when an object is created for a different user, it doesn't appear for another user, then I think it would be good to add the asserts to check for this user as well in addition to what you currently have.

Reply:
The plan was to check that it only returned ingredients for the specific user yeah, since at one point it was just returning all custom ingredients. I'll add asserts for the second user too 👍

Reply:
Yep, that's a good plan with checking by adding the second user!

Reply:
done 👍